_posts/en/newsletters/2021-04-14-newsletter.md

```diff
@@ -0,0 +1,225 @@
+    miners.  That generally allows reviewers to make the simplifying
+    assumption that height will only ever increase, making it easier to
+    analyze height-based activation mechanisms than MTP mechanisms.
+
+    These tradeoffs between the two proposals, among other concerns,
+    created an impasse that some developers thought was preventing either
+    PR from receiving additional review and, ultimately, getting one of
+    them merged into Bitcoin Core.  That impasse was resolved to the
+    satisfaction of some participants in the activation discussion when
+    the authors of the two PRs agreed to a compromise:
+
+    1. To use MTP for the earliest time when nodes begin counting blocks
+       signaling for the soft fork.  This is identical to the way
+       [BIP9][] versionbits and [BIP148][] UASF started counting blocks
+       for the soft forks they helped activate.
+
+    2. To use MTP for the *earliest* time when nodes stop counting block
+       signaling for the soft fork, with counting continuing until the
+       end of the 2,016-block retarget period that reached the stop time.
+       This is a change from BIP9 and BIP148 where MTP was used for the
+       *latest* time when counting stopped.  The difference removes the
+       ability for an activation attempt to go directly from *started*
+       to *failed*, simplifying analysis and guaranteeing that there
+       will be at least one complete 2,016 block period where miners can
+       signal for activation.
+
+    3. To use height for the minimum activation parameter.  This further
+       simplifies analysis and also remains compatible with the goal of
+       allowing multiple test networks to share activation parameters.
+       Even though height may differ on those networks, they can all use
+       a minimum activation height of `0` to activate within the window
+       defined by MTP.
+
+    Although some discussion participants expressed their displeasure
+    with the compromise proposal, its [implementation][bitcoin core
+    #21377] has now received reviews or
+    expressions of support from over a dozen active contributors to
+    Bitcoin Core and the maintainers of two other full node
+    implementations (btcd and libbitcoin).  We hope this momentum
+    to activate taproot continues and we'll be able to report additional
+    progress in a future newsletter.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Introduce deploymentstatus][review club
+#19438] is a PR ([#19438][Bitcoin Core #19438]) by Anthony Towns that proposes
+three helper functions to make it easier to [bury future deployments without
+changing all the code paths][easier burying] that check [BIP9][] status:
```

Comment:
```suggestion
changing all the code paths][easier burying] that check a soft fork's activation status:
```

(the point is to make these functions generic so they don't just apply to bip9 deployments - they also apply to buried deployments)